app/scripts/lib/constants.js

```diff
@@ -81,7 +81,10 @@ define(function (require, exports, module) {
     ACCESS_TYPE_ONLINE: 'online',
     ACCESS_TYPE_OFFLINE: 'offline',
 
-    DEFAULT_XHR_TIMEOUT_MS: 2500
+    DEFAULT_XHR_TIMEOUT_MS: 2500,
+
+    // TODO: Run this past rfeeley, do we want to show the user a message?
+    SIGN_UP_EXISTING_USER_SUCCESS: 'Account already exists, we signed you in instead!'
```

Comment:
You know...  I've gone back and forth about using named constants for strings too. If you want a named constant, can you put it in strings.js?